ui/hooks/src/use-comments.new.ts

```diff
@@ -170,12 +170,13 @@ export function useCreateComment() {
 }
 
 // hook used for editing a comment
-export function useEditComment(commentID: string) {
+export function useEditComment(commentID: string, isComment: boolean) {
   const sdk = getSDK();
   const queryClient = useQueryClient();
 
   return useMutation(
     (comment: IPublishData & { postID: string }) => {
+      if (!isComment) return;
```

Comment:
The use case for this flag is to prevent posting empty data?

Reply:
Yes, throws errors in the console otherwise

Reply:
I would suggest to rename this into something more verbose for the use case, like `hasCommentData`

Reply:
the return should be consistent, `return Promise.resolve()`